app/coffee/HistoryManager.coffee

```diff
@@ -21,16 +21,17 @@ module.exports = HistoryManager =
 				error = new Error("track changes api returned a failure status code: #{res.statusCode}")
 				return callback(error)
 
-	FLUSH_EVERY_N_OPS: 50
-	pushUncompressedHistoryOps: (project_id, doc_id, ops = [], callback = (error) ->) ->
+	FLUSH_EVERY_N_OPS: 100
+	pushUncompressedHistoryOps: (project_id, doc_id, ops = [], length, callback = (error) ->) ->
```

Comment:
This method is also wrongly named now. Not sure what it should be though. `recordAndFlushHistoryOps`?